recipes/botan/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
 find_package(botan REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} botan::botan)
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 20)
```

Comment:
@lieser `ERROR: botan/2.19.3 (test package): Error in build() method, line 26` the upgrade of compiler requirements is true for Botan 3 onwards. But building `test_package` for older version should not have this requirement otherwise it breaks the build.